docs/book/src/cronjob-tutorial/testdata/project/cmd/main.go

```diff
@@ -124,34 +122,22 @@ func main() {
 		tlsOpts = append(tlsOpts, disableHTTP2)
 	}
 
-	// Create watchers for metrics and webhooks certificates
-	var metricsCertWatcher, webhookCertWatcher *certwatcher.CertWatcher
-
 	// Initial webhook TLS options
 	webhookTLSOpts := tlsOpts
+	webhookServerOptions := webhook.Options{
+		TLSOpts: webhookTLSOpts,
```

Comment:
Coould we not pass directly 
```suggestion
		TLSOpts: tlsOpts,
```